pkg/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,30 @@
+// healthcheck package provides service to get grpc server health status.
+package healthcheck
+
+import (
+	"context"
+
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+// heath check for grpc server.
+type HealthChecker struct{}
+
+// check status and returns grpc healthcheckresponse
+func (s *HealthChecker) Check(ctx context.Context, req *grpc_health_v1.HealthCheckRequest) (*grpc_health_v1.HealthCheckResponse, error) {
+	return &grpc_health_v1.HealthCheckResponse{
+		Status: grpc_health_v1.HealthCheckResponse_SERVING,
+	}, nil
+}
+
+// stream the server status change.
+func (s *HealthChecker) Watch(req *grpc_health_v1.HealthCheckRequest, server grpc_health_v1.Health_WatchServer) error {
+	return server.Send(&grpc_health_v1.HealthCheckResponse{
+		Status: grpc_health_v1.HealthCheckResponse_SERVING,
+	})
+}
+
+// request to check the grpc server health.
```

Comment:
Typically a comment is started with the Name of the func, struct, or method.
see https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Fixed.